app/src/main/java/edu/wisc/ece/pinpoint/MainActivity.java

```diff
@@ -46,9 +51,24 @@ public void onCreate(Bundle savedInstanceState) {
             }
         });
 
-        // Fetch logged in user profile on app load
-        FirebaseDriver firebase = FirebaseDriver.getInstance();
-        firebase.fetchUser(firebase.getCurrentUser().getUid());
+
+        Handler handler = new Handler(Looper.getMainLooper());
+        handler.postDelayed(new Runnable() {
+                                @Override
+                                public void run() {
+
+                                    PeriodicWorkRequest saveRequest =
+                                            new PeriodicWorkRequest.Builder(PinNotificationActivity.class, 15, TimeUnit.MINUTES)
+                                                    // Constraints
+                                                    .build();
+
+                                    WorkManager work = WorkManager.getInstance(getApplicationContext());
+                                    work.enqueue(saveRequest);
+                                }
+
+
+        },1);
```

Comment:
Are we able to only start the notifications once the app is closed? I don't think we want a periodic task going while the app is open.